tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -26,18 +26,11 @@ import scala.language.postfixOps
 import scala.concurrent.duration.Duration
 import scala.concurrent.duration.DurationInt
 import scala.util.Random
-
 import org.apache.commons.io.FileUtils
 import org.junit.runner.RunWith
 import org.scalatest.junit.JUnitRunner
-
-import common.TestHelpers
-import common.TestUtils
+import common._
```

Comment:
Wouldn't import everything from common unless we actually use everything in that package.